docs/sources/operator/custom-resource-quickstart.md

```diff
@@ -170,6 +170,19 @@ spec:
         name: primary-credentials-metrics
         key: password
 
+  # As an alternative authencation method, Grafana Agent also supports OAuth2.
```

Comment:
@karengermond since you are working to rewrite this section, do we need this here?

Reply:
I'm thinking that this PR will close before I open mine.  So, we can keep it here.  I'm going to switch over to some Cloud integration work I have first.